src/openscm_runner/adapters/ciceroscm_adapter/ciceroscm.py

```diff
@@ -34,11 +34,14 @@ def _run(self, scenarios, cfgs, output_variables, output_config):
 
         cfgs is a list of indices to run
         """
+        LOGGER.info("Call to ciceroscm openscm-runner")
+        LOGGER.info(output_variables)
         if output_config is not None:
             raise NotImplementedError("`output_config` not implemented for Cicero-SCM")
 
         runs = run_ciceroscm_parallel(scenarios, cfgs, output_variables)
-
+        LOGGER.info("Returning from CICERO-SCM")
+        LOGGER.info(type(runs))
```

Comment:
```suggestion
        LOGGER.debug("Type of `runs`: %s", type(runs))
```

Reply:
So, these lines really aren't there because I plan to keep them around... I was just trying to figure out why the climate assessment pipeline doesn't work (or more specifically why my part isn't returning temperature variables) anymore, but now I can't even get the pipeline to import the from my commit at all anymore....